scripts/build_renewable_profiles.py

```diff
@@ -235,8 +236,11 @@
         excluder.add_raster(paths.corine, codes=codes, buffer=buffer, crs=3035)
 
     if "max_depth" in config:
-        func = lambda v: v <= -config['max_depth']
-        excluder.add_raster(paths.gebco, codes=func, crs=4236, nodata=-1000)
+        # lambda not supported for atlite + multiprocessing
+        # use named function np.greater with partially frozen argument instead
+        # to include only areas with depth > -max_depth
+        func = functools.partial(np.greater,-config['max_depth'])
+        excluder.add_raster(paths.gebco, codes=func, invert=True, crs=4236, nodata=-1000)
```

Comment:
```suggestion
        excluder.add_raster(paths.gebco, codes=func, crs=4236, nodata=-1000)
```
The filter does not need to be inverted. The new func returns the same as the old func, i.e. True for values which are below -config['max_depth']. Consider 
```
np.greater(5,3)
```  
-> True 

and likewise 
```
functools.partial(np.greater, 5)(3))
```
-> True
which is the same as 3 <= 5 (old func)

Reply:
Yes, you are right. Good catch. Thanks.